### ISSUE TITLE
Test set for FCN-8

### ISSUE THREAD

**Issue by kbarni** [YunYang1994/TensorFlow2.0-Examples#33]
The FCN-8 example uses the **VOC2007** and **VOC2012** datasets for training and the **VOC2007** dataset for evaluation. However the **VOC2012 train** dataset contains the **VOC2007 train and test** images. 

So basically you use a part of the train dataset to evaluate the network! No surprise that the results are nearly perfect!

With other images the results are really mediocre. Here's an example (the cyclist is segmented as bicycle(green) or dog (dark blue), the bike as person (pink) or car (cyan), the regions are quite random...)
![5](https://user-images.githubusercontent.com/26119925/68134770-e2c45780-ff22-11e9-8581-cdc6a0ab4922.jpg)

Still have to check the performance of FCN-8 with other implementations to see if it's an implementation problem, or this is the normal performance of the FCN-8 network.

**Comment by YunYang1994** [comment]
that maybe a mistake
